src/main/java/edu/cornell/kfs/concur/ConcurParameterConstants.java

```diff
@@ -10,6 +10,7 @@
     public static final String CONCUR_CUSTOMER_PROFILE_SUB_UNIT = "CONCUR_CUSTOMER_PROFILE_SUB_UNIT";
     public static final String DEFAULT_TRAVEL_REQUEST_OBJECT_CODE = "DEFAULT_TRAVEL_REQUEST_OBJECT_CODE";
     public static final String CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE = "CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE";
+    public static final String CONCUR_VALID_TRAVELER_STATUSES = "CONCUR_VALID_TRAVELER_STATUSES";
```

Comment:
I think we should name this such that it states these are valid concur status to be used in PDP with the employee payee type

Reply:
Ok, I'll rename the parameter accordingly based on the changes I make to the utility service method.